lib/middleware/index.js

```diff
@@ -25,19 +25,12 @@ Login.prototype.middleware = function(store, options) {
   if (typeof this.options.resetPasswordTimeLimit === 'string')
     this.options.resetPasswordTimeLimit = ms(this.options.resetPasswordTimeLimit);
 
-  this.store = store;
+  this.store = backend;
```

Comment:
I wonder where `this.store` is referenced, I'm not seeing it. Is this assignment really necessary?